app.py

```diff
@@ -50,7 +51,21 @@ def _get_local_time():
             return f"Could not process time information. {e}", 404
 
 
-#TODO Add a route that displays the current time for all US timezones.
+@app.route('/time-difference', methods=['GET', 'POST','PUT'])
+def time_diff():
+    try:
+        if request.args:
+            h = request.args.get('here')
+            t = request.args.get('there')
+
+        u = pytz.timezone('utc').localize(datetime.utcnow())
```

Comment:
Lines 61 to 63 are a little hard to follow. Can we use different variable names to describe the transformations that are happening?